README.md

```diff
@@ -28,11 +28,11 @@ The [master](https://github.com/butterproject/butter) branch which contains the
 If you encounter trouble with the above method, you can try:
 
 1. `npm install -g bower grunt-cli` (Linux: you may need to run with `sudo`)
-1. `cd desktop`
```

Comment:
this is only a possible location it can be anywhere and this is an example location